worlds/stardew_valley/logic.py

```diff
@@ -576,7 +576,8 @@ def __post_init__(self):
                                self.has("Crimsonfish") & self.has("Mutant Carp") & self.has("Legend"),
             "Danger In The Deep": self.can_mine_perfectly() & self.has_mine_elevator_to_floor(120),
             "Skull Cavern Invasion": self.can_mine_perfectly_in_the_skull_cavern() & self.has_max_buffs(),
-            "Qi's Prismatic Grange": self.has("Bug Meat") & self.can_spend_money(80000), # All colors can be bought except purple
+            "Qi's Prismatic Grange": self.has("Bug Meat") & self.can_spend_money(80000),
+            # All colors can be bought except purple
```

Comment:
hard disagree with this comment being below the relevant line lol, it really breaks readability. If it complains about line too long, we can newline on the &, or add the comment above instead. I personally don't care about line too long when it's a comment.

Reply:
Was automatically done, but sure.